fw/http_sess.c

```diff
@@ -259,7 +173,7 @@ tfw_http_sticky_build_redirect(TfwHttpReq *req, StickyVal *sv, RedirMarkVal *mv,
 	}
 
 	code = jsch_allow ? sticky->redirect_code: TFW_REDIR_STATUS_CODE_DFLT;
```

Comment:
Wiki https://tempesta-tech.com/knowledge-base/Sticky-Cookie/ says that the default response code is 503, but it actually 302. Please explain in the wiki why exactly 302 is used (please check the commits history and pull requests discussions) and when to prefer 3xx over 5xx and vice versa.

Reply:
I implement all changes in wiki after PR will be approved.